disdrodb/L0/readers/ARM/ARM_ld.py

```diff
@@ -228,4 +196,4 @@ def reformat_ARM_files(file_list, attrs):
     client = Client(processes=True)
 
     # Run the processing
-    main()
+    reader()
```

Comment:
I think we need to remove ` if __name__ == "__main__"` and the code below in all readers !
Also because without the click wrapper, no arguments will be passed to the function anymore

Reply:
ok remove these lines from the reader. This could be usefull while developing the reader, but you are right, they should not be in the production code.